File: JavaToCSharpConverter/Output/RescueSplitLine.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RescueJ
{
public class RescueSplitLine : RescueGeometryObject
{


  protected RescueSplitLine(long ndxIn)
  {
    nativeNdx = ndxIn;
  }

  public void dispose()
  {
    Delete_RescueSplitLine(nativeNdx);
  }

}

}

File: JavaToCSharpConverter/Source/RescueSplitLine.java
package org.Rescue.rjni;
import java.lang.*;
/*****************************************************************

    RescueSplitLine.java

    Rod Hanks   Sep  5 2005

******************************************************************/
public class RescueSplitLine extends RescueGeometryObject
{


  protected RescueSplitLine(long ndxIn)
  {
    nativeNdx = ndxIn;
  }

  public void dispose()
  {
    Delete_RescueSplitLine(nativeNdx);
  }

  private native void Delete_RescueSplitLine(long ndx);
}
